docs/concepts/gen.md

```diff
@@ -0,0 +1,20 @@
+# GEN script
+
+Perl scripts that read and process data from Perun into data files for the services (or SLAVE scripts for
+further processing). These scripts are run by the Engine.
+
+The file name must match the service name.
```

Comment:
This will not necessarily be true when we start using generic scripts (which is soon). I would change the wording a bit, the fact that gen/send script have to have the same file name seems more important to me

Reply:
Yes, I did not know if I should write it as it is currently, or how it will be after generic JSON and support for multiple scripts.